cpp/src/io/parquet/ipc/schema/Schema.fbs

```diff
@@ -243,9 +244,9 @@ table RunEndEncoded {
 }
 
 /// Exact decimal value represented as an integer value in two's
-/// complement. Currently only 128-bit (16-byte) and 256-bit (32-byte) integers
-/// are used. The representation uses the endianness indicated
-/// in the Schema.
+/// complement. Currently 32-bit (4-byte), 64-bit (8-byte),
+/// 128-bit (16-byte) and 256-bit (32-byte) integers are used.
+/// The representation uses the endianness indicated in the Schema.
```

Comment:
Does this need to be listed? Are there other options?

I'm assuming that this text is being copied verbatim by the generator into Schema_generated.h?

Reply:
> I'm assuming that this text is being copied verbatim by the generator into Schema_generated.h?

Yep, exactly. Comments updated from the same file in arrow repo.